Source/Moya.swift

```diff
@@ -4,6 +4,28 @@ import Result
 /// Closure to be executed when a request has completed.
 public typealias Completion = (result: Result<Moya.Response, Moya.Error>) -> ()
 
+public typealias ProgressBlock = (progress:ProgressResponse) -> Void
```

Comment:
For consistency with the rest of the code, I think we should add a space after `:` (`progress: ProgressResponse`).  Doesn't have to be part of this PR, we could clean up afterwards. 😃